string-manipulation-2/src/truncate.js

```diff
@@ -0,0 +1,8 @@
+/* exported truncate */
+function truncate(length, string) {
+  if (string.length <= length) {
+    return string + '...';
+  } else {
+    return string.slice(0, length) + '...';
```

Comment:
I think this will still work if you just had line 6 (the conditional isn't needed when you write it this way)